docs/source/usingmame/defaultkeys.rst

```diff
@@ -1,106 +1,558 @@
+**=** *(not numeric keypad)*
+    Volume Up for machines that have a volume control.
+**F1**
+    Power the machine on for machines that have specific power button behavior.
+**F2**
+    Power the machine off for machines that have specific power button behavior.
+
+    Service Mode button for machines that support that.
+**F3**
+    Soft resets the machine.
+**Shift+F3**
+    Performs a "hard reset", which tears everything down and re-creates it from
+    scratch. This is a more thorough and complete reset than the reset you get
+    from hitting F3.
+**LCtrl+F3**
+    Toggle uneven stretch. (*SDL ONLY*)
+**F4**
+    Shows the game palette, decoded graphics tiles/characters and any tilemaps.
+
+    Use the Enter key to switch between the three modes (palette, graphics, and
+    tilemaps).
+
+    Press F4 again to turn off the display.
+    The key controls in each mode vary slightly:
+
+    Palette/colortable mode:
+
+    * **[** **]** - switch between palette and colortable modes.
+    * **Up**/**Down** - scroll up/down one line at a time.
+    * **Page Up**/**Page Down** - scroll up/down one page at a time.
+    * **Home**/**End** - move to top/bottom of list.
+    * **-**/**+** - increase/decrease the number of colors per row.
+    * **Enter** - switch to graphics viewer.
+
+    Graphics mode:
+
+    * **[** **]** - switch between different graphics sets.
+    * **Up**/**Down** - scroll up/down one line at a time.
+    * **Page Up**/**Page Down** - scroll up/down one page at a time.
+    * **Home**/**End** - move to top/bottom of list.
+    * **Left**/**Right** - change color displayed.
+    * **R** - rotate tiles 90 degrees clockwise.
+    * **-**/**+** - increase/decrease the number of tiles per row.
+    * **Enter** - switch to tilemap viewer.
+
+    Tilemap mode:
+
+    * **[** **]** - switch between different tilemaps.
+    * **Up**/**Down**/**Left**/**Right** - scroll 8 pixels at a time.
+    * **Shift+Up**/**Down**/**Left**/**Right** - scroll 1 pixel at a time.
+    * **Control+Up**/**Down**/**Left**/**Right** - scroll 64 pixels at a time.
+    * **R** - rotate tilemap view 90 degrees clockwise.
+    * **-**/**+** - increase/decrease the zoom factor.
+    * **Enter** - switch to palette/colortable mode.
+
+    Note: Not all games have decoded graphics and/or tilemaps.
+**LCtrl+F4**
+    Toggles keeping aspect ratio.
+    (*SDL MAME only*)
+**LCtrl+F5**
+    Toggle Filter. (*SDL ONLY*)
+    (*SDL MAME only*)
+**Alt+Ctrl+F5**
+    Toggle HLSL Post-Processing.
+    (*Windows non-SDL MAME only*)
+**F6**
+    Toggle cheat mode. (if started with "-cheat")
+**LCtrl+F6**
+    Decrease Prescaling.
+**LCtrl+F7**
+    Increase Prescaling.
+**F7**
+    Load a save state. You will be prompted to press a key or select from the
+    menu to determine which save state you wish to load.
+
+    *Note that the save state feature is not supported for a large number of
+    drivers. If support is not enabled for a given driver, you will receive a
```

Comment:
This is unclear – it’s always enabled, it’s just not necessarily supported/known to work.  You get the warning if it isn’t marked as expected to work.

Reply:
Adjusted. You may want further adjustments on this one.